addons/mail/models/res_users_settings_volumes.py

```diff
@@ -14,6 +14,26 @@ class ResUsersSettingsVolumes(models.Model):
     guest_id = fields.Many2one('res.partner', ondelete='cascade', index=True)
     volume = fields.Float(default=0.5, help="Ranges between 0.0 and 1.0, scale depends on the browser implementation")
 
+    def _mail_users_settings_volume_format(self):
+        records = []
```

Comment:
This doesn't contain records directly, but records data (or formatted records), so I would change this name if possible.

Also it would be better to use the actual record name (volume_setting or something like that, instead of a generic "record" variable).